celery/app/base.py

```diff
@@ -459,7 +459,9 @@ def _task_from_fun(self, fun, name=None, base=None, bind=False, **options):
             try:
                 task.__qualname__ = fun.__qualname__
             except AttributeError:
+                logger.error('_task_from_fun: AttributeError inside %s.', name)
```

Comment:
Are we sure the celery logger is active early enough for these to get logged correctly?  Just remembering those issues reported in celery's GitHub issue tracker about some logging during initialization being silently ignored, which led us to also printing everything just in case.

Reply:
I did't see this entry in logs. Also added print now.